cita-auth/src/handler.rs

```diff
@@ -1090,3 +1094,24 @@ fn snapshot_response(sender: &Sender<(String, Vec<u8>)>, ack: Resp, flag: bool)
         ))
         .unwrap();
 }
+
+// min quota is 21000 (or 53000 if tx.to == 0) + (4 per byte)
+// tx quota: 21000
+// create contract: 53000
+// only verify if tx.version > 2
+pub fn verify_min_quota(tx: &Transaction) -> bool {
+    match tx.get_version() {
```

Comment:
Actually, it's no need to consider the version at cita-auth module

Reply:
This function is used in block verify too. In early block, it's need to consider the version.

Reply:
I see.